src/main/java/in/ekstep/am/builder/TokenDetails.java

```diff
@@ -0,0 +1,13 @@
+package in.ekstep.am.builder;
+
+public interface TokenDetails {
+    void setAccess_token(String access_token);
+    void setExpires_in(long expires_in);
+    void setRefresh_expires_in(long refresh_expires_in);
```

Comment:
Review comment - All variables and methods to be in camel casing and not snake casing